_extras/guide.md

```diff
@@ -245,13 +245,20 @@ stack_selection.columns = stack_selection.columns.droplevel()
 
 ### Challenge - Changing Types
 
+* Try converting the column `plot_id` to floats using `surveys_df.plot_id.astype("float")`. Next try converting `weight` to an integer. What goes wrong here? What is Pandas telling you?
```

Comment:
Next -> then

Reply:
`Then try converting `weight` column to an integer type`

Reply:
I'd remove `What goes wrong here?`... and say `What does _pandas_ tell you when you try doing that?` but I'll leave it to @wrightaprilm to decide what is the best wording here

Reply:
How about "What error message does Pandas give you? What does this error mean?" Emphasizing specifically that we want them to read and comprehend the output, which in this case is very helpful and informative!

Reply:
Thanks @maxim-belkin  and @wrightaprilm. Is this a good wording to commit?

```suggestion
* Try converting the column `plot_id` to floats using `surveys_df.plot_id.astype("float")`. Then, try converting the contents of the `weight` column to an integer type. What error message does Pandas give you? What does this error mean?
```

Reply:
I like this wording!